server/package.json

```
@@ -33,7 +33,7 @@
     "sync:open-api": "node ./dist/bin/sync-open-api.js",
     "sync:sql": "node ./dist/bin/sync-sql.js",
     "email:dev": "email dev -p 3050 --dir src/emails",
-    "postinstall": "[[ \"$npm_config_global\" != \"true\" ]] && patch-package || true"
+    "postinstall": "[ \"$npm_config_global\" != \"true\" ] && patch-package || true"
```

```suggestion
    "postinstall": "[ \"$npm_config_global\" = \"true\" ] || patch-package”
```

is there a reason the current implementation logic is so strange?

---

I added the `|| true` to the end of it to not exit with a failed code if the install was global. Your implementation is much more concise than my pass at it.

---

Oops, thanks for fixing the quote